app/controllers/api/v1/ping_things_controller.rb

```diff
@@ -11,14 +11,26 @@ class InvalidRecordCount < StandardError; end
       def index
         limit = [(index_params[:limit] || 240).to_i, 9999].min
         page = index_params[:page] || 1
+        with_total_count = index_params[:with_total_count].to_s == "true" ? true : false
```

Comment:
```suggestion
        with_total_count = [true, "true"].include? index_params[:with_total_count]
```

Reply:
I cant's see how it's better. to_s will do exatly the same thing and having only one "true" in this statement is much easier to read imo.